x-pack/plugins/lens/public/datasources/text_based/types.ts

```diff
@@ -23,6 +23,7 @@ export interface TextBasedField {
 export interface TextBasedLayer {
   index: string;
   query: AggregateQuery | undefined;
+  table?: Datatable;
```

Comment:
Why are we adding this here Peter?

Reply:
i added support for `render only lens` to this already, this type is all i had to bring in from the other PR.